lnst/Recipes/ENRT/ConfigMixins/EnableIntelSSTTFMixin.py

```diff
@@ -11,7 +11,7 @@
 class EnableIntelSSTTFMixin(BaseSubConfigMixin):
     TURBO_PATH = "/sys/devices/system/cpu/intel_pstate/no_turbo"
 
-    enable_intel_sst_tf = BoolParam(default=False)
+    enable_intel_sst_tf = BoolParam()
```

Comment:
since the entire `EnableIntelSSTTFMixin` is added in the previous commit can we squash the changes to it from this commit into the previous one?

i'd keep this commit only to add the mixin to the `BaremetalEnrtCommonMixins` class dependencies.